tests/sqllogictests/suites/base/09_fuse_engine/09_0026_merge_into.test

```diff
@@ -901,6 +901,23 @@ WHEN NOT MATCHED THEN
     INSERT (order_id, user_id, order_type, asset_type, quantity, price, status, created_at, updated_at)
     VALUES ((SELECT MAX(order_id) FROM orders) + 1, synthetic_orders.user_id, synthetic_orders.synthetic_order_type, synthetic_orders.asset_type, synthetic_orders.total_quantity, 0, 'pending', synthetic_orders.synthetic_date, synthetic_orders.synthetic_date);
 
+statement ok
+create table tb_01 (id int,c1 varchar,c2 datetime(0),c3 json);
+
+statement ok
+create table tmp_01 like tb_01;
+
+statement ok
+insert into tmp_01 values(1,'abc',now(),parse_json('{"a":1}'));
```

Comment:
[SQL] select id,c1,to_date(c2),c3 from tb_01;
[Diff] (-expected|+actual)
-   1 abc 2023-11-29 {"a":1}
+   1 abc 2023-11-30 {"a":1}
at tests/sqllogictests/suites/base/09_fuse_engine/09_0026_merge_into.test:916

Reply:
oh, sorry, this test is bad.